package.json

```diff
@@ -124,5 +124,6 @@
       "./cypress/main.ts",
       "./cypress/support/step_definitions/*.ts"
     ]
-  }
+  },
+  "packageManager": "yarn@1.22.21+sha256.dbed5b7e10c552ba0e1a545c948d5473bc6c5a28ce22a8fd27e493e3e5eb6370"
```

Comment:
Did this sneak in with the yarn/pnpm investigation you've been doing?

Reply:
Argh, I need to address this in my local environment. I updated corepack and since this newest version, every single `yarn` command I run adds this (not just `yarn install`, `yarn dev`, too, etc.). I’ll remove it here.

Maybe we can add the `packageManager` entry though? It seems like there might be a bit of standardization happening here. In any case, it doesn’t belong here, so I’ll remove it.

Reply:
Done.

Reply:
No worries. Yeah definitely could add this later. I read `corepack` is potentially being moved to stable soon